src/zkp/pisch.rs

```diff
@@ -138,6 +138,7 @@ impl Proof for PiSchProof {
         // Verifier samples e in F_q
         let challenge = positive_challenge_from_transcript(transcript, &k256_order())?;
         if challenge != self.challenge {
+            dbg!("xoiiiiiinicht");
```

Comment:
Just leaving a comment here, pretty sure this will get cleaned up

Reply:
done